Kconfig

```diff
@@ -5,4 +5,4 @@
 # as the module Kconfig entry point (see zephyr/module.yml). You can browse
 # module options by going to Zephyr -> Modules in Kconfig.
 
-rsource "*/Kconfig"
+rsource "soc/Kconfig"
```

Comment:
While it is currently empty, I believe we should also include `boards/Kconfig` (or drop board directly, but I believe check_compliance.py wants it).